tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1862,6 +1862,56 @@ error: Cannot instantiate type \\"BucketProps\\" because it is a struct and not
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`super_call.w 1`] = `
+"error: Cannot call super method because we're not in an instance method (\\"super\\" is unknown in this context)
+   --> ../../../examples/tests/invalid/super_call.w:32:7
+   |
+32 | super.do();
+   |       ^^ Cannot call super method because we're not in an instance method (\\"super\\" is unknown in this context)
```

Comment:
Can we add `super` as a reserved keyword?

Reply:
it's already there (https://github.com/winglang/wing/blob/ce24a6a4df52f1e4bff80c3363ab00105abbefb1/libs/wingc/src/parser.rs#L95). Do you mean something else?

Reply:
Ok cool 👍